src/OmniSharp.Abstractions/Models/ICanBeEmptyResponse.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace OmniSharp.Models
+{
+    public interface ICanBeEmptyResponse
```

Comment:
Could we mark this interface as `internal`? The outside world doesn't need to know about this (technically). This way we can avoid a major breaking change. (well kind of anyway)

Reply:
Made ICanBeEmptyResponse interface internal.